### ISSUE TITLE
Sample Images

### ISSUE THREAD

**Issue by gabynordic** [Mohitsharma44/pyfrac#3]
Do you have any sample images to work with?

**Comment by Mohitsharma44** [comment]
@gabynordic Sure.. try this sample image 
![south-1528409296 41](https://user-images.githubusercontent.com/5817360/42718645-c33c9ed6-86d8-11e8-9b75-a152ab432d9d.jpg)
I am not maintaining the modules in `convert` directory but they should still work (although highly unoptimized). Let me know if you face issues.

**Comment by gabynordic** [comment]
I have the convert script running, but am getting some very unusual values in the CSV output files for my images (~4000 Degrees Celsius). Any recommendations?

**Comment by Mohitsharma44** [comment]
Are you converting the above image? or some other image?

**Comment by gabynordic** [comment]
![flir](https://user-images.githubusercontent.com/40034288/42897986-6a75b6fe-8a87-11e8-80dd-3a40ee6e1ec1.jpg)
It is a different image

**Comment by Mohitsharma44** [comment]
Yeah, so that *may* be different because different sensors have different formulae for extracting temperature from sensor readouts.
This code has been written with FLIR A310 and A320 (or basically cameras that use ICDA320 sensor)

**Comment by gabynordic** [comment]
Ahh, that make sense. Thanks for your help! Do you have any recommendations as to how to find these formulas for other sensors?

**Comment by Mohitsharma44** [comment]
The only way I was able to do this was because I found the spec sheet of
the sensor. So I would suggest look up the documentation for your camera
and find the sensor name,make,model etc and look it up on Google.
Also, I don't think reverse engineering would work because the relationship
between sensor readout and actual temperature in degrees is not linear
(afaik) and Flir is generally very tight lipped about the formulas they use
(in their tools like Flir tools +)


Mohit Sharma
https://sharmamohit.com
